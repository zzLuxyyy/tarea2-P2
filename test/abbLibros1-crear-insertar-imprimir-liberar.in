crearABBLibrosVacio
imprimirABBLibros
liberarABBLibros

crearABBLibrosVacio
crearFecha 01/10/1995
crearLibro 843397030 "Ensayo sobre la ceguera" "Jos√©" "Saramago" "Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano" 2
insertarLibroABBLibros
imprimirABBLibros

liberarABBLibros
Fin
