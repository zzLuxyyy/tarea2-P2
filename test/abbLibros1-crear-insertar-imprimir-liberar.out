1>Se ejecutó crearTABBLibrosVacio exitosamente.
2>3>4>Se ejecutó crearTABBLibrosVacio exitosamente.
5>6>7>Se insertó el libro con ISBN 843397030 al árbol.
8>Libro 843397030
Título: Ensayo sobre la ceguera
Autor: José Saramago
Descripción: Una epidemia de ceguera azota a una ciudad, revelando lo peor del ser humano
Género: 2
Fecha de edición: 1/10/1995
9>10>Fin.
